docs/changelog.rst

```diff
@@ -22,6 +22,12 @@ New features
 
 .. _`Issue 86`: https://github.com/berkerpeksag/astor/issues/86
 
+Bug fixes
+~~~~~~~~~
+
+* Fix function, class and module docstrings for Python 3.7
```

Comment:
This looks like a changelog for #102, right?

Reply:
Oops. My bad. Fixed.